src/main/java/com/derongan/minecraft/mineinabyss/world/AbyssWorldManagerImpl.kt

```diff
@@ -34,7 +34,13 @@ class AbyssWorldManagerImpl(config: Configuration) : AbyssWorldManager {
         val layerName = map[NAME_KEY] as String
         val subHeader = map[SUB_KEY] as String
 
-        val layer = LayerImpl(layerName, subHeader, numLayers++, deathMessage = " ${map.getOrDefault(DEATH_MESSAGE_KEY, "in $layerName")}")
+        val layer = LayerImpl(layerName, subHeader, numLayers++, deathMessage = " ${map.getOrDefault(DEATH_MESSAGE_KEY, "in $layerName")}",
+                maxCurseMultiplier = map["maxCurseMultiplier"] as? Float ?: 2f,
+                minCurseMultiplier =  map["minCurseMultiplier"] as? Float ?: 0.5f,
+                maxCurseRadius = map["maxCurseRadius"] as? Float ?: 1000f,
+                minCurseRadius = map["minCurseRadius"] as? Float ?: 2000f,
+                curseOverrideRegions = emptyList()
```

Comment:
I assume this will be implemented later by someone?

Reply:
yes waiting on #37